prime-router/src/test/kotlin/SubmissionReceiverTests.kt

```diff
@@ -301,6 +301,11 @@ class SubmissionReceiverTests {
         "42&ISO||445297001^Swab of internal nose^SCT^^^^2.67||||53342003^Internal nose structure (body structure)^" +
         "SCT^^^^2020-09-01|||||||||202108020000-0500|20210802000006.0000-0500"
 
+    val bulk_fhir_bundles = """
```

Comment:
In FhirConverterTests.kt we're reading FHIR bundles in from external files. I think it makes the test easier to read. Would it be possible to switch the rest of the tests over to using that strategy as well?

Reply:
Moved test data to external files.